run.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
-cd llama.cpp
-make llama-server
-cd ..
 
-export LLAMA_N_CTX=32000
+if nvcc --version
+then
+  export CUDA_VISIBLE_DEVICES=0
```

Comment:
Does 0 mean all? Or the first one, which is index 0?

Reply:
first one, this is only for local dev anyway

Reply:
Ah right.